tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/mapper/CurriculumMembershipMapper.java

```diff
@@ -63,6 +81,16 @@ public List<ProgrammeMembershipDTO> allEntityToDto(
       }
       programmeMembershipDto.getCurriculumMemberships()
           .add(curriculumMembershipToCurriculumMembershipDto(curriculumMembership));
+      if (programmeMembershipDto.getUuid() != null) {
+        Optional<ConditionsOfJoining> conditionsOfJoiningOptional
+            = conditionsOfJoiningRepository.findById(programmeMembershipDto.getUuid());
+        if (conditionsOfJoiningOptional.isPresent()) {
+          programmeMembershipDto.setConditionsOfJoining(
+              conditionsOfJoiningMapper.toDto(conditionsOfJoiningOptional.get()));
+        } else {
+          programmeMembershipDto.setConditionsOfJoining(null);
+        }
+      }
```

Comment:
Can I get a quick check on whether this is needed given the call on L#77, which calls the code you've added :point_down:?

I've removed and the test for this method passes.

Reply:
Ugh, thanks Pepe. Yes, completely unnecessary code 🤦